src/SortableContainer/index.js

```diff
@@ -339,12 +341,21 @@ export default function SortableContainer(WrappedComponent, config = {withRef: f
 				if (transitionDuration) {
 					node.style[`${vendorPrefix}TransitionDuration`] = `${transitionDuration}ms`;
 				}
+
+				const margin = getElementMargin(node);
+				// margins merge so we subtract them
+				const delta = (
+					axis === 'x'
+					? this.width  + this.margin.left + this.margin.right  - margin.left
+					: this.height + this.margin.top  + this.margin.bottom - margin.top
+				);
+
```

Comment:
Negative margins work as expected :smile: 
The only supported feature is margin collapsing which happens on vertical margins only (top and bottom) and when they are of the same type (positive or negative)